website/docs/d/iam_access_group_template_versions.html.markdown

```diff
@@ -0,0 +1,114 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_iam_access_group_template_version"
+description: |-
+  Get information about ibm_iam_access_group_template_version
+subcategory: "IAM Access Groups"
+---
+
+# ibm_iam_access_group_template_version
+
+Provides a read-only data source to retrieve information about an ibm_iam_access_group_template_version. You can then reference the fields of the data source in other resources within the same configuration by using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_iam_access_group_template_versions" "ibm_iam_access_group_template_versions" {
+	template_id = "template_id"
```

Comment:
Can we some sample value or some interpolation syntax to read "template_id"

Reply:
updated